src/store/validation/validationHelpers.ts

```diff
@@ -89,13 +89,27 @@ export function debugValidationStatus(status: IValidationStatus, customMessage:
   }
 }
 
-function getErrorMessageLocalizationKey(keyPrefix: string, ajvErrorObject: ErrorObject) : string {
+function getErrorMessageLocalizationKeyForProperty(keyPrefix: string, ajvErrorObject: ErrorObject) : string {
   const entity = keyPrefix.startsWith('flow') ? 'flows' : 'blocks'
-  const property = ajvErrorObject.dataPath.replaceAll('/', '-')
+  const property = ajvErrorObject.dataPath
+    .replaceAll('/', '-')
+    // Replacing digits with zeros to eliminate resource indexes
+    .replaceAll(/\d/g, '0')
```

Comment:
let's replace them with `x` not `0`, to have something like `'base.x-times'` localization key